types/newrelic.d.ts

```diff
@@ -8,25 +8,12 @@ export function recordCustomEvent(eventType: string, eventName: string, attribut
 export function startInteraction(actionName: string, cb: Function, fail: any): Promise<any>;
 export function endInteraction(interactionId: string, cb: any, fail: any): void;
 export function sendConsole(type: any, args: any): void;
-export function sendConsole(type: any, args: any): void;
-export function send(name: any, args: any): void;
 export function send(name: any, args: any): void;
 /**
- * Records JavaScript errors for cordova.
- * @param {string} name The name of the error.
- * @param {string} message The message of the error.
- * @param {string} stack The error stack of the error.
- * @param {boolean} isFatal The flag for whether the error is fatal.
- */
-export function recordError(name: string, message: string, stack: string, isFatal: boolean, cb: any, fail: any): void;
-/**
- * Records JavaScript errors for cordova.
- * @param {string} name The name of the error.
- * @param {string} message The message of the error.
- * @param {string} stack The error stack of the error.
- * @param {boolean} isFatal The flag for whether the error is fatal.
+ * Records JavaScript errors for Cordova.
+ * @param {Error} err The name of the error.
  */
-export function recordError(name: string, message: string, stack: string, isFatal: boolean, cb: any, fail: any): void;
+export function recordError(err: Error, cb: any, fail: any): void;
```

Comment:
Technically, the error can be a string, but I just defined it as Error only here